resources/views/list-view/sorting.blade.php

```diff
@@ -0,0 +1,36 @@
+{{-- list-view.sorting.blade
+
+Renders the dropdown for the sorting select
+To customize it you should shange the UI component used, YOU SHOULD NOT CUSTOMIZE ANYHING HERE
+UI components used:
+  - form/select --}}
+<div class="leading-tight">
+  {{-- Sorting dropdown --}}
+  @component('laravel-views::components.drop-down')
+    @slot('trigger')
+      @include('laravel-views::components.buttons.select', [
+      'title'=> $sortableBy->flip()->get($sortBy) ?? __('Sort By'),
```

Comment:
it'd be great if the button title says something like `Sort by: {sortOrder} {field}` or something like that, once you have selected something, it becomes unclear what the button is for, what do you think?